<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>T3_AF_faceLog</class>
 <widget class="QDialog" name="T3_AF_faceLog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="_backgroundLabel_">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>450</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(:/Pictures/faceLog_background.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="_nameLineEdit_">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>68</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;border-width:0;border-style:outset;color:white</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="_ageLineEdit_">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>127</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;border-width:0;border-style:outset;color:white</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="_imageAddressLineEdit_">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>247</y>
     <width>123</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;border-width:0;border-style:outset;color:white</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="_addNewUserPushBtn_">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>250</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(:/Pictures/faceLog_addNew.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="_checkPushBtn_">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>392</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(:/Pictures/faceLog_button.png);color:white</string>
   </property>
   <property name="text">
    <string>查找</string>
   </property>
  </widget>
  <widget class="QPushButton" name="_clearPushBtn_">
   <property name="geometry">
    <rect>
     <x>705</x>
     <y>392</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(:/Pictures/faceLog_button.png);color:white</string>
   </property>
   <property name="text">
    <string>清空</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="_searchByNameLineEdit_">
   <property name="geometry">
    <rect>
     <x>565</x>
     <y>340</y>
     <width>210</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(:/Pictures/faceLog_textEdit.png);color:white</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="_searchByNameLabel_">
   <property name="geometry">
    <rect>
     <x>515</x>
     <y>340</y>
     <width>50</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white</string>
   </property>
   <property name="text">
    <string>姓名</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="_imageLabel_">
   <property name="geometry">
    <rect>
     <x>515</x>
     <y>242</y>
     <width>50</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white</string>
   </property>
   <property name="text">
    <string>图片</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="_userTypeLabel_">
   <property name="geometry">
    <rect>
     <x>515</x>
     <y>182</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white</string>
   </property>
   <property name="text">
    <string>注册类型</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="_ageLabel">
   <property name="geometry">
    <rect>
     <x>515</x>
     <y>122</y>
     <width>50</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white</string>
   </property>
   <property name="text">
    <string>年龄</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="_nameLabel_">
   <property name="geometry">
    <rect>
     <x>515</x>
     <y>63</y>
     <width>50</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white</string>
   </property>
   <property name="text">
    <string>姓名</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="_addUserBackgroundLabel_">
   <property name="geometry">
    <rect>
     <x>505</x>
     <y>0</y>
     <width>295</width>
     <height>271</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(:/Pictures/faceLog_addUserBackground.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="_userImageLabel_">
   <property name="geometry">
    <rect>
     <x>669</x>
     <y>9</y>
     <width>93</width>
     <height>140</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="_dateTimeLabel_">
   <property name="geometry">
    <rect>
     <x>312</x>
     <y>426</y>
     <width>190</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white</string>
   </property>
   <property name="text">
    <string>_dateTimeLabel_</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
   </property>
  </widget>
  <widget class="QPushButton" name="_exitPushBtn_">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>2</y>
     <width>29</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(:/Pictures/faceLog_back.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="_getImagePushBtn_">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>250</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(:/Pictures/faceLog_image.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="_userTypeComboBox_">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>180</y>
     <width>171</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color:black;
</string>
   </property>
  </widget>
  <widget class="QTableView" name="_faceInfoTableView_">
   <property name="geometry">
    <rect>
     <x>33</x>
     <y>33</y>
     <width>439</width>
     <height>389</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;border-width:0;border-style:outset;color:white</string>
   </property>
  </widget>
  <widget class="QLabel" name="_battery_">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>426</y>
     <width>49</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
   </property>
  </widget>
  <widget class="QLabel" name="_battIMG_">
   <property name="geometry">
    <rect>
     <x>225</x>
     <y>426</y>
     <width>133</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>_backgroundLabel_</zorder>
  <zorder>_userImageLabel_</zorder>
  <zorder>_addUserBackgroundLabel_</zorder>
  <zorder>_nameLineEdit_</zorder>
  <zorder>_ageLineEdit_</zorder>
  <zorder>_imageAddressLineEdit_</zorder>
  <zorder>_addNewUserPushBtn_</zorder>
  <zorder>_checkPushBtn_</zorder>
  <zorder>_clearPushBtn_</zorder>
  <zorder>_searchByNameLineEdit_</zorder>
  <zorder>_searchByNameLabel_</zorder>
  <zorder>_imageLabel_</zorder>
  <zorder>_userTypeLabel_</zorder>
  <zorder>_ageLabel</zorder>
  <zorder>_nameLabel_</zorder>
  <zorder>_dateTimeLabel_</zorder>
  <zorder>_exitPushBtn_</zorder>
  <zorder>_getImagePushBtn_</zorder>
  <zorder>_userTypeComboBox_</zorder>
  <zorder>_faceInfoTableView_</zorder>
  <zorder>_battery_</zorder>
  <zorder>_battIMG_</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
